subworkflows/local/bam_variant_calling_haplotypecaller/main.nf

```diff
@@ -126,31 +126,34 @@ workflow BAM_VARIANT_CALLING_HAPLOTYPECALLER {
 
         realigned_bam = BAM_MERGE_INDEX_SAMTOOLS.out.bam_bai
 
-        VCF_VARIANT_FILTERING_GATK(haplotypecaller_vcf.join(haplotypecaller_tbi),
-                    fasta,
-                    fasta_fai,
-                    dict,
-                    intervals_bed_combined,
-                    known_sites_indels.concat(known_sites_snps).flatten().unique().collect(),
-                    known_sites_indels_tbi.concat(known_sites_snps_tbi).flatten().unique().collect())
-
-        filtered_vcf = VCF_VARIANT_FILTERING_GATK.out.filtered_vcf.map{ meta, vcf-> [
-                [
-                    patient:meta.patient,
-                    sample:meta.sample,
-                    status:meta.status,
-                    sex:meta.sex,
-                    id:meta.sample,
-                    num_intervals:meta.num_intervals,
-                    variantcaller:"haplotypecaller"
-                ],
-                vcf
-            ]
+        if (!(skip_tools && skip_tools.split(',').contains('haplotypecaller_filter'))) {
+
+            VCF_VARIANT_FILTERING_GATK(haplotypecaller_vcf.join(haplotypecaller_tbi),
+                        fasta,
+                        fasta_fai,
+                        dict,
+                        intervals_bed_combined,
+                        known_sites_indels.concat(known_sites_snps).flatten().unique().collect(),
+                        known_sites_indels_tbi.concat(known_sites_snps_tbi).flatten().unique().collect())
+
+            filtered_vcf = VCF_VARIANT_FILTERING_GATK.out.filtered_vcf.map{ meta, vcf-> [
```

Comment:
For the output of the sw: nono-filtered, if it is skipped? and then use just hc output for annotation?

Reply:
yes, that's what I was thinking about when I mentioned the csv file